src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -66,6 +72,32 @@ public GridStorageCache(final IGrid g) {
     public void onUpdateTick() {
         this.itemMonitor.onTick();
         this.fluidMonitor.onTick();
+
+        this.itemBytesTotal = 0;
+        this.itemBytesUsed = 0;
+        try {
+            for (ICellProvider icp : this.activeCellProviders) {
+                if (icp.getClass() == Class.forName("appeng.tile.storage.TileDrive")) {
+                    // All Item Cell
+                    for (IMEInventoryHandler meih : icp.getCellArray(StorageChannel.ITEMS)) {
+                        if (((DriveWatcher<IAEItemStack>) meih).getInternal().getClass()
```

Comment:
wrap with `instanceof CellInventoryHandler handler` you will have casted variable for free. And replace all usages